tests/Backend/Workspaces/SynapseWorkspacesUnloadTest.php

```diff
@@ -95,6 +95,56 @@ public function testCreateTableFromWorkspaceWithInvalidColumnNames()
         }
     }
 
+    public function testImportFromWorkspaceWithInvalidTableName()
+    {
+        // create workspace and source table in workspace
+        $workspace = $this->initTestWorkspace();
+
+        $tableId = 'Languages3';
+        $tableIdInvalid = '[Language3]';
+
+        $backend = WorkspaceBackendFactory::createWorkspaceBackend($workspace);
+        $backend->dropTableIfExists($tableId);
+
+        $connection = $workspace['connection'];
+
+        $db = $this->getDbConnection($connection);
+
+        $quotedTableId = $db->getDatabasePlatform()->quoteIdentifier(sprintf(
+            '%s.%s',
+            $connection['schema'],
+            $tableId
+        ));
+
+        $db->query("create table $quotedTableId (
+			[Id] integer not null,
+			[Name] varchar(50) not null,
+			[_update] varchar(50) not null
+		);");
+        $db->query("insert into $quotedTableId ([Id], [Name], [_update]) values (1, 'cz', 'x');");
+        $db->query("insert into $quotedTableId ([Id], [Name], [_update]) values (2, 'en', 'z');");
+
+        $table = $this->_client->apiPost("buckets/" . $this->getTestBucketId(self::STAGE_IN) . "/tables", array(
+            'dataString' => 'Id,Name',
+            'name' => 'languages',
+            'primaryKey' => 'Id',
+        ));
```

Comment:
Tohle tu vůbec nemusí být IMHO

Reply:
Ta cast s `$db->query` tu byt nemusi.
Ale ten `apiPost` jo, pac potrebuju potom `$table['id']` do volani `writeTableAsyncDirect`.

Kdyz si to `table['id']` necham vygenerovat, tak to skonci na jiny chybe:
> storage.tables.notFound
> The table "Languages3" was not found in the bucket "in.c-API-tests-1e762931bcd0740bf445815b75fa5c89f8001949" in the project "201"

Reply:
Jo jasně! Tak ale není potřeba mít tam tu vytvořenou tabulku ve workspacu, ne?

Reply:
Jo, ono už to zmizlo v tom navazujícím commitu jasně ;)